ui/src/app/pages/workspace/workspace-about.tsx

```diff
@@ -109,17 +109,17 @@ export const WorkspaceAbout = fp.flow(withUserProfile(), withUrlParams(), withCd
     };
   }
 
-  async componentDidMount() {
+  componentDidMount() {
```

Comment:
This was the source of most of the puppeteer failures. I'm surprised that this was working previously - but it was definitely highly dependent on arbitrary React rendering order.

Specifically, it was breaking the rules of React state:

```
this.setState({workspace});
...
// expecting this.state to have been synchronously updated
f(this.state.workspace);
```

The other changes in the PR must have somehow resulted in an ordering change, possibly this was always a risk but a race condition was removed.